app/Http/Controllers/Service/CentreController.php

```diff
@@ -135,6 +134,10 @@ public function exportRegistrationsSummary()
             $rows[] = $row;
         }
 
+        usort($rows, function ($a, $b) {
```

Comment:
Return 0 if values on either side are equal
Return 1 if value on the left is greater
Return -1 if the value on the right is greater

I had to look it up... worth a comment with ^^ ?